common/changes/office-ui-fabric-react/sayali-arrowFocusFix_2018-03-15-21-28.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Add shouldInputLoseFocusOnArrowKey callback for scenario to determine if input should loose focus when arrow key is present when Tabbing is enabled on All elements or Input elements",
```

Comment:
As a consumer I can't make sense of the reasoning here. Can you fix some of the language? This comment goes into release notes (and it can be markdown, so use backticks around property names.)

FocusZone: adding `shouldInputLoseFocusOnArrowKey` optional callback for scenarios where the user press an arrow key on an input element, but we would want to disable moving focus away.

Honestly, I don't quite think that's your scenario, but reading this it is very difficult to understand why as a user I would need to use this callback.

Reply:
Oh good to know. Thanks David. I have updated the change log.